riak/src/main/java/com/yahoo/ycsb/db/package-info.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2014 Basho Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The YCSB binding for <a href="http://basho.com/products/riak-kv/">Riak KV</a>
+ * 2.0+.
+ */
+package com.yahoo.ycsb.db;
```

Comment:
Could we put this in a package named `com.yahoo.ycsb.db.riak`? I'm trying to get the datastore bindings more separated.